apps/api/src/queries/buildViewModelForResource/viewModels/event-sourced-vocabulary-list.view-model.ts

```diff
@@ -54,7 +55,14 @@ export class EventSourcedVocabularyListViewModel
         // TODO we should serialize when returning from the query service automatically so we can use instances here if we'd like
         view.name = buildMultilingualTextWithSingleItem(textForName, languageCodeForName).toDTO();
 
-        view.actions = [];
+        view.actions = [
+            'PUBLISH_RESOURCE',
+            'CREATE_NOTE_ABOUT_RESOURCE',
+            'CONNECT_RESOURCES_WITH_NOTE',
+            'TRANSLATE_VOCABULARY_LIST_NAME',
+            'ADD_TERM_TO_VOCABULARY_LIST',
+            'REGISTER_VOCABULARY_LIST_FILTER_PROPERTY',
```

Comment:
Again, nonsensical question but this wouldn't include `ANALYZE_TERM_IN_VOCABULARY_LIST`?

Reply:
> No, the user doesn't need to see the option to analyze a term if there are no terms and no vocabulary lists have been analyzed.

Right!  Sorry, missed some of the context there